### ISSUE TITLE
crash: wallet.get_full_history - unsupported operand type(s) for -

### ISSUE THREAD

**Issue by SomberNight** [spesmilo/electrum#3954]
On git master 99710099fb3566b2c4c1a976e8cf79d431ad56db.

```
Traceback (most recent call last):
  File "/home/user/wspace/electrum/gui/qt/__init__.py", line 217, in start_new_window
    w = self.create_window_for_wallet(wallet)
  File "/home/user/wspace/electrum/gui/qt/__init__.py", line 179, in create_window_for_wallet
    w = ElectrumWindow(self, wallet)
  File "/home/user/wspace/electrum/gui/qt/main_window.py", line 201, in __init__
    self.load_wallet(wallet)
  File "/home/user/wspace/electrum/lib/util.py", line 275, in <lambda>
    return lambda *args, **kw_args: do_profile(func, args, kw_args)
  File "/home/user/wspace/electrum/lib/util.py", line 271, in do_profile
    o = func(*args, **kw_args)
  File "/home/user/wspace/electrum/gui/qt/main_window.py", line 344, in load_wallet
    self.history_list.update()
  File "/home/user/wspace/electrum/gui/qt/util.py", line 484, in update
    self.on_update()
  File "/home/user/wspace/electrum/lib/util.py", line 275, in <lambda>
    return lambda *args, **kw_args: do_profile(func, args, kw_args)
  File "/home/user/wspace/electrum/lib/util.py", line 271, in do_profile
    o = func(*args, **kw_args)
  File "/home/user/wspace/electrum/gui/qt/history_list.py", line 202, in on_update
    r = self.wallet.get_full_history(domain=self.get_domain(), from_timestamp=self.start_timestamp, to_timestamp=self.end_timestamp, fx=fx)
  File "/home/user/wspace/electrum/lib/util.py", line 275, in <lambda>
    return lambda *args, **kw_args: do_profile(func, args, kw_args)
  File "/home/user/wspace/electrum/lib/util.py", line 271, in do_profile
    o = func(*args, **kw_args)
  File "/home/user/wspace/electrum/lib/wallet.py", line 1042, in get_full_history
    start_balance = out[0]['balance'].value - out[0]['value'].value
TypeError: unsupported operand type(s) for -: 'NoneType' and 'int'
```

**Comment by SomberNight** [comment]
(similar conditions as https://github.com/spesmilo/electrum/issues/3952, while syncing)

**Comment by ecdsa** [comment]
fixed in f3440f5a20bee6c64f738ffc32010e94ad8c6dab
